src/content/docs/en/guides/authentication.mdx

```diff
@@ -188,6 +188,78 @@ const session = await getSession(Astro.request);
 - [`auth-astro` on GitHub](https://github.com/nowaythatworked/auth-astro?tab=readme-ov-file#auth-astro)
 - [Auth.js documentation](https://authjs.dev/)
 
+## Clerk
+
+Clerk is a complete suite of embeddable UIs, flexible APIs, and admin dashboards to authenticate and manage your users. An [official SDK](https://clerk.com/docs/references/astro/overview) for Astro is available.
+
+### Installation
+
+Install `@clerk/astro` using the package manager of your choice.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install @clerk/astro
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add @clerk/astro
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add @clerk/astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Configuration
+
+Follow the [Astro Quickstart](https://clerk.com/docs/quickstarts/astro) guide to set up Clerk integration and middleware in your Astro project.
+
+### Usage
+
+Protecting your pages in the client
```

Comment:
```suggestion
Clerk provides templating components that allow you to control the visibility of pages based on your visitor's authentication state. Show logged out users sign in button instead of content available to users who are logged in:
```

Reply:
I made a small change here. Let me know if that works!